.github/workflows/discovery.yaml

```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    - cron: '0 12 * * TUE'
+  pull_request: # REMOVE BEFORE MERGING
+  workflow_dispatch:    
+name: Update Discovery Generated Types
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
```

Comment:
Do these get updated by dependabot or renovate bot? Wanting to make sure we're not adding a manual maintenance step of updating